package.json

```diff
@@ -2,8 +2,9 @@
   "name": "st_pax_tracker",
   "private": true,
   "dependencies": {
-    "datatables.net": "^1.10.15",
```

Comment:
I think it would be a good idea to keep this dependency explicit

Reply:
Since I can't edit that file directly, how exactly does that work? Do I yarn add (thing-at-specific-version-number)?

Reply:
>  ... I can't edit that file directly ...

Oh?

Reply:
I thought it was like a Gemfile.lock! It's not?

Reply:
Like I definitely _didn't_ edit it directly this time, let's just put it that way.

Reply:
`Gemfile` : `package.json` :: `Gemfile.lock` : `yarn.lock`

Reply:
Oooh.